src/cashflow.ts

```diff
@@ -9,32 +11,39 @@ const defaultRefillFactor = 20;
 const defaultRefillThresholdFactor = 8;
 
 /** The amount of tokens that will be sent to the user */
-export function creditAmount(token: TokenTicker): number {
+export function creditAmount(token: TokenTicker, factor: number = 1): Amount {
+  // Load this from connection?
+  const fractionalDigits = Number.parseInt(process.env.FAUCET_FRACTIONAL_DIGITS || "0", 10);
   const amountFromEnv = process.env[`FAUCET_CREDIT_AMOUNT_${token}`];
-  if (amountFromEnv) {
-    return Int53.fromString(amountFromEnv).toNumber();
-  } else {
-    return 10;
-  }
+  const wholeNumber = amountFromEnv ? Int53.fromString(amountFromEnv).toNumber() : 10;
+  const total = wholeNumber * factor;
+  // replace BN with BigInt with TypeScript 3.2 and node 11
+  const quantity = new BN(total).imul(new BN(10 ** fractionalDigits)).toString();
```

Comment:
`new BN(10 ** fractionalDigits)` may break for Ethereum as 10^18 is out of max safe integer range.

But what works is `new BN("1" + "0".repeat(fractionalDigits))` or `new BN(10).pow(new BN(fractionalDigits))`